src/main/java/gregtech/loaders/oreprocessing/ToolRecipeHandler.java

```diff
@@ -178,22 +157,22 @@ public static void processStick(OrePrefix stickPrefix, Material material, ToolPr
                     "PPf", "PP ", "Sh ",
                     'S', new UnificationEntry(OrePrefix.stick, material),
                     'P', new UnificationEntry(OrePrefix.plate, material));
-        }
-
-        if (material.hasFlags(GENERATE_PLATE, GENERATE_ROD, GENERATE_BOLT_SCREW)) {
-            ModHandler.addShapedRecipe(String.format("wire_cutter_%s", material.toString()),
-                    MetaItems.WIRE_CUTTER.getStackForm(material),
-                    "PfP", "hPd", "STS",
-                    'S', new UnificationEntry(stickPrefix, material),
-                    'P', new UnificationEntry(OrePrefix.plate, material),
-                    'T', new UnificationEntry(OrePrefix.screw, material));
 
-            ModHandler.addShapedRecipe(String.format("branch_cutter_%s", material.toString()),
-                    MetaItems.BRANCH_CUTTER.getStackForm(material),
-                    "PfP", "PdP", "STS",
-                    'S', new UnificationEntry(stickPrefix, material),
-                    'P', new UnificationEntry(OrePrefix.plate, material),
-                    'T', new UnificationEntry(OrePrefix.screw, material));
+            if (material.hasFlags(GENERATE_BOLT_SCREW)) {
```

Comment:
does this need `GENERATE_PLATE` as well?

Reply:
Plate is in the outer if statement.